nav_msgs/Path curPath
bool ifFirstPoint
---

bool success
